src/components/DogBattle.js

```diff
@@ -1,19 +1,28 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Dog from "./Dog.js";
 import "./DogBattle.css";
 
-class DogBattle extends React.Component {
-  render() {
-    return (
-      <div className="DogBattle">
-        <h2 className="DogBattle-title">Choose the best dog</h2>
-        <div className="DogBattle-images">
-          <Dog image="http://via.placeholder.com/300x300" />
-          <Dog image="http://via.placeholder.com/300x300" />
-        </div>
+const DogBattle = () => {
+  const [images, setImages] = useState([]);
+  useEffect(() => {
+    fetch(`https://dog.ceo/api/breeds/image/random/2`)
+      .then((res) => res.json())
+      .then((data) => {
+        if (data) {
+          const values = Object.values(data.message);
```

Comment:
Perhaps we could use `dogImageArray` instead of `values`, so we know what is being retrieved.